lib/webauthn/attestation_statement/packed.rb

```diff
@@ -70,7 +71,16 @@ def attestation_certificate
         attestation_certificate_chain&.first
       end
 
-      def valid_certificate_chain?(credential)
+      def valid_certificate_chain?
+        if attestation_certificate_chain
+          attestation_certificate_chain[1..-1].all? { |c| certificate_in_use?(c) }
+        else
+          true
+        end
+      end
+
+      # TODO: Reevaluate this check
```

Comment:
Good call - certificates containing RSA keys should be valid too.

Reply:
Yes.